MineSweeper/MineSweeper.cs

```diff
@@ -34,18 +51,18 @@ public GameStatus PlayTurn(int x, int y, TurnType turnType)
             {
                 TurnType.DiscoverCell => PlayTurnDiscover(cell),
                 TurnType.ToggleFlag => PlayTurnFlag(cell),
-                _ => throw new ArgumentException("Invalid Turn Type")
+                _ => throw new ArgumentException($"Unknown Turn Type '{turnType}'.")
             };
 
             if (gameStatus != GameStatus.InProgress)
             {
                 return gameStatus;
             }
 
-            return CheckWin();
+            return CheckIsWin();
```

Comment:
možná `IsWinOrStillInProgress()`

Reply:
Jj, tohle by se možná ještě mohlo trochu líp pojmenovat nebo upravit, o přesně to CheckWin dělá